src/pages/Content/ActionList.tsx

```diff
@@ -25,7 +26,9 @@ function ActionListItem({
           (action.selectors.text?.length ?? -1) < 75 ? (
             <span>"{action.selectors.text}"</span>
           ) : (
-            <span className="mono">{getBestSelectorForAction(action)}</span>
+            <span className="mono">
+              {getBestSelectorForAction(action, ScriptType.Playwright)}
```

Comment:
Should we pass scriptType as prop to ActionList component so we don't harcode the value here ?

Reply:
Same as above, I think maybe the long term solution is to have a preferred library the user chooses to be saved in local storage.